kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -137,6 +141,12 @@ private PartitionedTableId getRecordTable(SinkRecord record) {
 
     TableId baseTableId = topicsToBaseTableIds.get(record.topic());
 
+    BigQuery bigQuery = getBigQuery();
```

Comment:
It feels kind of out of place to have this autocreation happen in `getRecordTable` can you add a separate `maybeCreateTable` method and call that after `getRecordTable`?

Reply:
Got it, updated accordingly!